app/pages/stacking/delegated-stacking/components/choose-delegator-stx-address.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC } from 'react';
+import { useFormik } from 'formik';
+
+import { stxAddressValidator } from '@utils/validators/stx-address-validator';
+
+import { StackingStepBaseProps } from '../../utils/abstract-stacking-step';
+import {
+  StackingStep as Step,
+  StackingStepAction as Action,
+  StackingStepDescription as Description,
+} from '../../components/stacking-form-step';
+import { CryptoAddressForm } from '../../components/crypto-address-form';
+
+interface ChooseDelegatorStxAddressStepProps extends StackingStepBaseProps {
+  description: string;
+  value?: string;
+  onEdit(): void;
+  onComplete(address: string): void;
+}
+
+export const ChooseDelegatorStxAddressStep: FC<ChooseDelegatorStxAddressStepProps> = props => {
+  const { isComplete, description, state, step, title, value, onEdit, onComplete } = props;
```

Comment:
:~)

Reply:
Some of this is indeed prop drilling to pass to next el down. I guess this could use a context? 🤔   I don't see much issue being explit here though.